pkg/policies/engine/rego/rego.go

```diff
@@ -67,8 +69,8 @@ func (r *Rego) Verify(ctx context.Context, policy *engine.Policy, input []byte,
 	// add module
 	regoFunc := rego.ParsedModule(parsedModule)
 
-	// add query. Note that the predefined rule to look for is `deny`
-	query := rego.Query(fmt.Sprintf("%v.deny\n", parsedModule.Package.Path))
+	// add query. Note that the predefined rule to look for is `violations`
+	query := rego.Query(fmt.Sprintf("%v.%s\n", parsedModule.Package.Path, mainRule))
```

Comment:
should we have some retro-compat until all the policies have been written? 

Basically document the new format but allow `deny` if it's already set?

Reply:
sure, I've added retro compatibility.